sign_language_segmentation/bin.py

```diff
@@ -110,21 +130,33 @@ def main():
 
     for tier_id, segments in tiers.items():
         eaf.add_tier(tier_id)
+        frames_per_millisecond = fps*1000
         for segment in segments:
-            eaf.add_annotation(tier_id, int(segment["start"] / fps * 1000), int(segment["end"] / fps * 1000))
+            # convert frame numbers to millisecond timestamps, for Elan
+            start_frame_time = int(segment["start"] / fps * frames_per_millisecond)
```

Comment:
this implementation is now wrong. 
Was:
```py
segment["start"] / fps * 1000
```
Now:
```py
segment["start"] / fps * fps * 1000
```

Reply:
I suggest 
```py
start_time = int(segment["start"] / fps * 1000)
```

Reply:
Ah, it was wrong indeed! Fixed now